src/sentry/features/manager.py

```diff
@@ -263,6 +263,34 @@ def batch_has(
                 feature_names, actor, projects=projects, organization=organization
             )
         else:
+            # Fall back to default handler if no entity handler available. We can add batching here
+            # if w
+            project_features = filter(lambda name: name.startswith("projects:"), feature_names)
+            if projects and project_features:
+                results: MutableMapping[str, Mapping[str, bool]] = {}
+                for project in projects:
+                    proj_results = results[f"project:{project.id}"] = {}
+                    for feature_name in project_features:
+                        proj_results[feature_name] = self.has(feature_name, project, actor=actor)
+                return results
+
+            org_features = filter(lambda name: name.startswith("organizations:"), feature_names)
+            if organization and org_features:
+                org_results = {}
+                for feature_name in org_features:
+                    org_results[feature_name] = self.has(feature_name, organization, actor=actor)
+                return {f"organization:{organization.id}": org_results}
+
+            unscoped_features = filter(
+                lambda name: not name.startswith("organizations:")
+                and not name.startswith("projects:"),
```

Comment:
@wedamija what about `user:` features?

Reply:
I know it's not working for Flagr yet but if we ever add it this would definitely be a landmine

Reply:
Tbh I don't want to handle more cases that are already unhandled - I've been pulled pretty far off course by various things around features, just need to get this fixed and move forward.